apps/web/src/design-system/template-button/ChannelButton.tsx

```diff
@@ -109,12 +115,23 @@ export function ChannelButton({
   const disabledColor = disabled ? { color: theme.colorScheme === 'dark' ? colors.B40 : colors.B70 } : {};
   const disabledProp = disabled ? { disabled: disabled } : {};
   const { classes: popoverClasses } = usePopoverStyles();
+  const viewport = useViewport();
+
+  const {
+    watch,
+    formState: { errors },
+  } = useFormContext();
 
-  const { watch } = useFormContext();
+  let showErrors = initialErrors;
```

Comment:
This one sounds like a boolean to me, maybe we can rename this to: `formErrorsContent` or similar?